### ISSUE TITLE
Глава 6, не отображаются шаблоны аккаунтов

### ISSUE THREAD

**Issue by alpacas9** [DiscoverMeteor/DiscoverMeteor_Ru#130]
После добавления {{loginButtons}} в header шаблон не отображаются кнопки Login в правом углу. Вместо них ошибка -  No login services configured. В консоли 2 ошибки: 

```
Uncaught TypeError: Cannot read property 'push' of undefined
Uncaught Error: There are multiple templates named 'layout'. Each template needs a unique name.
```

Собственно этот вопрос подымался http://stackoverflow.com/questions/29970680/meteor-uncaught-typeerror-cannot-read-property-push-of-undefined/29971598#29971598 , но решения не найдено.

**Comment by avalanche1** [comment]
`accounts-password` package добавил?

**Comment by alpacas9** [comment]
Да, добавил. Делал точь-в-точь по главе, пакеты успешно установились. Проблема не решилась.

**Comment by avalanche1** [comment]
С авторским repo для 6 главы сравнивал?

**Comment by xamedow** [comment]
Была та же проблема, поставил ian:accounts-ui-bootstrap-3 вместо mrt, все заработало.
